tests/py/dynamo/conversion/test_embedding_bag_aten.py

```diff
@@ -22,29 +75,181 @@ class TestEmbeddingBagConverter(DispatchTestCase):
                 padding_idx=-1,
             ),
             param(
-                test_name="1d_indices_2",
-                weight=torch.randn((10, 3), dtype=torch.float32),
-                indices=torch.tensor([1, 2, 4, 5, 4, 3], dtype=torch.int32),
-                offsets=torch.tensor([0, 5], dtype=torch.int32),
+                test_name="1d_indices_6",
+                weight=torch.randn((10, 4), dtype=torch.float32),
+                indices=torch.tensor([1, 2, 4, 5, 4, 3, 2, 9], dtype=torch.int32),
+                offsets=torch.tensor([0, 5, 5], dtype=torch.int32),
+                scale_grad_by_freq=False,
+                mode=2,
+                sparse=False,
+                per_sample_weights=None,
+                include_last_offset=False,
+                padding_idx=-1,
+            ),
+            param(
+                test_name="1d_indices_7",
+                weight=torch.randn((10, 4), dtype=torch.float32),
+                indices=torch.tensor([1, 2, 4, 5, 4, 3, 2, 9], dtype=torch.int32),
+                offsets=torch.tensor([0, 8, 8], dtype=torch.int32),
                 scale_grad_by_freq=False,
                 mode=0,
                 sparse=False,
-                per_sample_weights=torch.randn((6,)),
+                per_sample_weights=None,
+                include_last_offset=True,
+                padding_idx=-1,
+            ),
+            param(
+                test_name="1d_indices_8",
+                weight=torch.randn((10, 4), dtype=torch.float32),
```

Comment:
Could one more test case be added or an existing one modified to try `weight` having dtype `torch.half`? I noticed some of the constant tensors are instantiated as `np.float32` and want to verify this wouldn't be an issue

Reply:
Do you mean some tensors are instantiated as `torch.float16`?

Reply:
Yes, that's right - trying out with the weight tensors, specifically